### PR TITLE
Add /api/add test for positive and negative numbers

### PR DESCRIPTION
**Pull request by epamsaratovjs**

### CODE REVIEW

File: src/app.test.js [hunk 1]
```diff
@@ -29,5 +29,19 @@ describe('GET /api/add', () => {
                 expect(text).toBe(result.toString());
             });
     });
+
+    it('Add positive and negative numbers', async () => {
```
Comment (epamsaratovjs) [inline src/app.test.js:33]: Looks good to me

**Review by vitalii-antonov — APPROVED** [review]
